src/Dapr.Extensions.Configuration/DaprSecretStoreConfigurationExtensions.cs

```diff
@@ -44,6 +44,34 @@ public static class DaprSecretStoreConfigurationExtensions
             return configurationBuilder;
         }
 
+        /// <summary>
+        /// Adds an <see cref="IConfigurationProvider"/> that reads configuration values from the Dapr Secret Store.
+        /// </summary>
+        /// <param name="configurationBuilder">The <see cref="IConfigurationBuilder"/> to add to.</param>
+        /// <param name="store">Dapr secret store name.</param>
+        /// <param name="metadata">The secrets metadata.</param>
+        /// <param name="client">The Dapr client</param>
+        /// <returns>The <see cref="IConfigurationBuilder"/>.</returns>
+        public static IConfigurationBuilder AddDaprSecretStore(
+            this IConfigurationBuilder configurationBuilder,
+            string store,
+            Dictionary<string, string> metadata,
```

Comment:
It would be really great if we could make metadata optional here - since usually there's not going to be anything in there. If this is a required arg, then everyone has to stop, look it up, figure out they don't care, and then move with their life :)

Reply:
Great point 👍

Done!